src/cloudabi-run/cloudabi-run.1

```diff
@@ -110,6 +110,10 @@ or the special values
 .Dq Li stdout
 and
 .Dq Li stderr .
+.It Cm "tag:nuxi.nl,2015:cloudabi/executable: int"
+Provides a file descriptor of the target executable
+.Ar path ,
+opened O_EXEC if possible, O_RDONLY otherwise.
```

Comment:
Could you replace this with:

    opened
    .Dv O_EXEC
    if possible,
    .Dv O_RDONLY
    otherwise.

The `.Dv` properly formats it. Putting a newline after a comma is also common tradition to make diffs smaller when making changes.

Reply:
Before I forget: also bump the man page's date!